src/core/hle/JVS/JVS.cpp

```diff
@@ -429,7 +429,7 @@ DWORD WINAPI XTL::EMUPATCH(JvsNodeReceivePacket)
 	// Receive the packet from the connected IO board
 	uint8_t DeviceId = g_pJvsIo->GetDeviceId();
 
-	uint16_t payloadSize = g_pJvsIo->ReceivePacket(&Buffer[6]);
+	uint16_t payloadSize = uint16_t(g_pJvsIo->ReceivePacket(&Buffer[6]));

```

Comment:
This is an odd syntax for a cast;
If using C style casts, it should be formatted as `(uint16_t)g_pJvsIo->ReceivePacket(&Buffer[6])` or if using a C++ style cast, use reinterpret_cast or similar.

Reply:
I'll fix it